pom.xml

```diff
@@ -52,23 +52,41 @@
 	<dependency>
 	    <groupId>org.apache.cxf</groupId>
 	    <artifactId>cxf-bundle</artifactId>
-	    <version>2.7.10</version>
+	    <version>3.0.0-milestone2</version>
```

Comment:
Is there a particular reason why you're using a milestone version of cxf-bundle? It looks like the financials POM is using a non-milestone CXF version, but that POM is also using finer-grained CXF JARs instead of cxf-bundle.